src/main/java/edu/greenblitz/tobyDetermined/commands/telescopicArm/elbow/DebugRotateBy10Degrees.java

```diff
@@ -0,0 +1,10 @@
+package edu.greenblitz.tobyDetermined.commands.telescopicArm.elbow;
```

Comment:
Dont think its only for debug

Reply:
I think it's OK, if we'll need it we have it in our repo and I think it's good enough